bin/rake

```diff
@@ -4,8 +4,7 @@ begin
 rescue LoadError => e
   raise unless e.message.include?('spring')
 end
-# frozen_string_literal: true
 
-require_relative "../config/boot"
-require "rake"
+require_relative '../config/boot'
+require 'rake'
```

Comment:
I think Rubocop won't like the single quotes.

Reply:
I did run rubocop and it was surprisingly fine with this! I tried to keep whatever files were generated as-is to reduce future diffs, because the slight modifications that had been done to generated files in the past (removing comments, changing quote types) made it hard to see what was actually changed during the upgrade this time around.